Documentation/devel/architecture.md

```diff
@@ -83,6 +83,13 @@ Specifically, it must:
   - systemd/nspawn: a cgroup/namespace based isolation environment using systemd, and systemd-nspawn.
   - kvm: a fully isolated kvm environment.
 
+There are also out of tree stage1:
+- [stage1-xen](https://github.com/rkt/stage1-xen), stage1 based on the Xen hypervisor
+- [volo](https://github.com/lucab/rkt-volo), stage1 written in Rust, akin to stage1-fly
+- [docker-skim](https://github.com/coreos/docker-skim), TODO: not clear to me what it is
```

Comment:
For reference: this is a weaker version of stage1-fly, which gets rid of the chroot by bundling only some dependencies and then letting `run` inject `LD_PRELOAD` and other PATH-related tricks.